spec/models/listing_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Listing, type: :model do
-  let(:listing) { build(:offer) }
+  let(:listing) { build(:offer, person: create(:person), service_area: create(:service_area)) }
```

Comment:
Won't FactoryBot do this automatically?

Reply:
It is def supposed to! But I noticed our tests were adding these things so I was thinking it needed to follow suit. 

FactoryBot is also supposed to provide shortcuts where you can list the assn name only (don't have to use longer syntax of `association :foo`), but it seemed to be needed to get some tests to pass so I went with it. Def could circle back on this.

Reply:
@exbinary yeah, the "extra" person declaration does seem to be needed for this to work, but for something more odd -- it *can* pass without service_area.